sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedFluxBase.java

```diff
@@ -68,8 +68,8 @@ public PagedFluxBase(Supplier<Mono<P>> firstPageRetriever) {
      */
     public PagedFluxBase(Supplier<Mono<P>> firstPageRetriever,
                          Function<String, Mono<P>> nextPageRetriever) {
-        Objects.requireNonNull(firstPageRetriever, "First page supplier cannot be null");
-        Objects.requireNonNull(nextPageRetriever, "Next page retriever function cannot be null");
+        Objects.requireNonNull(firstPageRetriever, "First page supplier cannot be null.");
```

Comment:
nit: This can be made consistent with the rest to use the variable name instead of the description.
```suggestion
        Objects.requireNonNull(firstPageRetriever, "'firstPageRetriever' cannot be null.");
```

Reply:
Sure, Thank you fixing it now :)